docs/devops-guidelines/Configuration-Management.md

```diff
@@ -0,0 +1,39 @@
+[DevOps Strategy](/Project-Artifacts/DevOps-Strategy)
+
+**Capabilities Inventory**
+
+|Capability|Ideal State|Tools In Inventory|
+|--|--|--|
+|Infrastructure Provisioning|Infrastructure is provisioned and configured as code along with the rest of the workload in an automated CI/CD pipeline|AWS CloudFormation, Azure Resource Manager|
+|Infrastructure Configuration|Infrastructure is provisioned and configured as code along with the rest of the workload in an automated CI/CD pipeline|AWS Config, AWS CLI, Azure Policy, Azure CLI|
+|Application Provisioning|Automatically create application CI in ServiceNow based on what gets deployed; users and administrators provisioned with proper access|ServiceNow, AzureAD, Okta|
+|Application Configuration|Configuration settings are exposed based on user roles.  Changes to configuration items are audited and sent through and appropriate approval process.|AWS OpWorks|
+
+**Infrastructure Provisioning**
+- Infrastructure will be provisioned using AWS CloudFormation templates.  This will allow the team to standardize on a tool several team members already know that the company is licensed for, has support for from AWS and will allow us to take advantage of all AWS native capabilities.  DCX is not intended to be cloud agnostic as it must take advantage of vendor specific services, e.g. CloudFront.
+
+- All CloudFormation templates will be stored in Azure DevOps and deployed use the Azure DevOps deployment pipeline.
+
+- Infrastructure provisioning will follow the [resource naming guidelines](https://phillips66.sharepoint.com/sites/IT_CloudRunway/DOH/Pages/Resource%20Naming.aspx) and [tagging guidelines](https://phillips66.sharepoint.com/sites/IT_CloudRunway/DOH/Pages/Resource%20Tags.aspx) outlined by CloudRunway.
```

Comment:
It looks like private links. Should this be omitted?

Reply:
Yes, looks like the incorrect guidelines were posted